guidelines-expectations.md

```diff
@@ -0,0 +1,86 @@
+
+ 9. **Expect and accept a lack of closure.** We will cover a myriad of topics throughout this workshop and not all questions will be answered or clear. 
+
+ 10. **We are all a work in progress.** We will not assume that one comment or one opinion made at one time captures the whole of a person's character. We are all in the process of learning, including the facilitators.
+
+ ## Expectations of event attendees  
+* Attendees are an active part of the workshop and help set the event atmosphere!
+* Have an open mind and treat others as they have something to offer.  
+* Be ready and willing to learn from instructors, helpers, and each other.
+* If you have the expertise, we invite others to contribute to the conversation, especially if directly related to their own experiences. This can be done through the stickies, group notes, telling an instructor/helper, or the google form.
+* Actively participate in events they attend  
+    * Use stickies
+    * Contribute to group notes
+    * If you have the capacity, help the people around you  
+* Ask questions if they have them  
+    * Ask helpers
+    * Type it in the etherpad chat
+    * Write it on a sticky at the end of each lesson
+    * Submit your question to the online form.
+* Respect others at the event
+* When receiving help from others during a lesson:  
+    * Use a whisper voice to lessen the impact on others around them. 
+    * Be open to "driving" the computer 
+    * If more extensive help is necessary, be open to the helper/instructor  
+* Avoid disruptive behavior including:
+    * Speaking to those around them during the lesson.
+* Be an active and responsive bystander by either:  
+   * Speaking up in the moment of an event.
+   * Submitting a climate response form.
+   * Talking to instructors and course coordinators regarding events that you may witness.
+
+ ## Expectations of instructors and helpers    
+* Have an open mind and treat others as they have something to offer.
+* Help event attendees learn and grow in a safe/brave and welcoming environment.   
+    * Welcome and encourage questions   
+    * When helping someone at their computer:  
+        * Be kind when approaching a learner at their computer and give them the benefit of a doubt that they are doing the best they can. 
+        * Be aware of the physical space between those that surround them.
+        * Prioritize the opportunity for the learner to "drive" the computer. 
+        * If it makes more sense for the instructor/helper to "drive" the learners computer, **always ask for permission** to touch someone else's computer.   
+    * Be an active and responsive by-stander by either:  
+        * Speaking up in the moment.
+        * Submitting a climate response form.
+        * Talking to instructors and course coordinators regarding events that you may witness.
+* Teach attendees.
```

Comment:
Let's add a sub-bullet! Maybe "Asking leading questions and helping learners get to answers on their own"